microsoft/testsuites/ltp/ltp.py

```diff
@@ -340,14 +345,31 @@ def _install(self) -> bool:
         self.node.tools[Mkdir].create_directory(top_src_dir, sudo=True)
         self.node.tools[Chmod].update_folder(top_src_dir, "a+rwX", sudo=True)
 
-        # clone ltp
-        git = self.node.tools[Git]
-        ltp_path = git.clone(
-            self.LTP_GIT_URL, cwd=PurePosixPath(top_src_dir), dir_name=top_src_dir
-        )
+        if self._tar_file_path:
+            self._log.debug(f"Use downloaded ltp tar: {self._tar_file_path}!")
+            self._remote_tar_path = self.get_tool_path(
```

Comment:
1. The `_remote_tar_path` is not used in other method, it doesn't need to be a class variable.
2. The name should be like `remote_source_file`.